examples/specs/line_color_halo.vl.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "https://vega.github.io/schema/vega-lite/v4.json",
+  "description": "Multi-series Line Chart with Halo. Use pivot and repeat-layer as a workaround to facet groups of lines and their halo strokes. See https://github.com/vega/vega-lite/issues/6192 for more discussion.",
+  "data": {"url": "data/stocks.csv"},
+  "transform": [{
+    "pivot": "symbol",
+    "value": "price",
+    "groupby": ["date"]
+  }],
+  "repeat": {
+    "layer": ["AAPL", "AMZN", "GOOG", "IBM", "MSFT"]
+  },
+  "spec": {
+    "layer": [{
+      "mark": {"type": "line", "stroke": "white", "strokeWidth": 4},
+      "encoding": {
+        "x": {"field": "date", "type": "temporal"},
```

Comment:
Do you want to pull x and y out of the individual layers?

Reply:
Just one encoding, probably not worth the additional complexity to read the code.